ui/lesson/TransactionsChallenge/index.tsx

```diff
@@ -394,7 +424,7 @@ const TransactionChallenge: FC<ITransactionProps> = ({
             <div className="max-md:gap-4 flex flex-col items-stretch justify-between md:h-14 md:flex-row">
               <div className="flex items-center align-middle transition duration-150 ease-in-out md:px-5">
                 <div className="font-nunito text-[21px] text-white opacity-50 transition duration-150 ease-in-out">
-                  {t('Lets move on to the first challenge!')}
+                  {t('Lets move on to the last challenge!')}
```

Comment:
Last challenge?

Reply:
swapped this as we start with a prefilled page and then have one page at the very end